index.html

```diff
@@ -86,11 +86,12 @@ <h2><b>Tonic</b></h2>
 
        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
      
-       <ul class="resources">
-        <li><a  class="button" href="#">HTML</a> <li>
-        <li><a  class="button" href="#">CSS</a> </li>
-        <li><a  class="button" href="#">javaScript</a></li>      
-      </ul>
+        <ul class="resources">
+          <li><a  class="button" href="#">HTML</a> <li>
+          <li><a  class="button" href="#">CSS</a> </li>
+          <li><a  class="button" href="#">javaScript</a></li>      
+        </ul>
+    
 
         <div class="action"> 
           <form class="normalbutton">
```

Comment:
- [ ] - [OPTIONAL ] Why have you chosen to use a form instead of a simple div as a container? I don't this is really a good way, you want to change it later, will you?

Reply:
Thank you for pointing this out. I will make it a habit to simply be using divs. :)